frontend/package.json

```diff
@@ -248,7 +248,7 @@
     "browser-env": "3.x",
     "cache-loader": "^4.1.0",
     "chalk": "2.4.x",
-    "chromedriver": "77.x",
+    "chromedriver": "90.x",
```

Comment:
You'll need to commit the yarn lockfile as well.

Reply:
Thank you for pointing out that I needed to commit yarn.lock, it fixed the frontend prow test but now e2e-gcp-console is failing. Is it something I can fix?
Thank you, 
Olga.

Reply:
@olavtar I suspect it's a test flake unrelated to your change :/